presto-hive/src/main/java/io/prestosql/plugin/hive/InternalHiveConnectorFactory.java

```diff
@@ -104,6 +109,7 @@ public static Connector createConnector(String catalogName, Map<String, String>
                         binder.bind(PageSorter.class).toInstance(context.getPageSorter());
                         binder.bind(CatalogName.class).toInstance(new CatalogName(catalogName));
```

Comment:
Just add `newSetBinder(binder, EventListener.class);` here. I don't see a need for `defaultEventListenerModule()` method.

Reply:
better reading experience along with expressing the intention of it, so that the reader doesn't have to guess that it's a necessary default Event Listener.

Reply:
It is a matter of taste. I would prefer not to over-optimize reading experience, but rather follow the current layout.

Reply:
Agreed, we should just bind an empty set here.